nuxeo-runtime/nuxeo-runtime-stream/src/main/java/org/nuxeo/runtime/avro/AvroSchemaDescriptor.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     pierre
+ */
+package org.nuxeo.runtime.avro;
+
+import org.nuxeo.common.xmap.annotation.XNode;
+import org.nuxeo.common.xmap.annotation.XObject;
+
+/**
+ * The avro descriptor.
+ * 
+ * @since 10.2
+ */
+@XObject("avroSchema")
```

Comment:
I disagree, I'd bet there is already at least one XObject("schema"), schema is so common you never know what it is about I feel this kind of naming a lot clearer.

Reply:
Yes but it doesn't matter, the XObjects don't live globally and there's no risk of confusion. This will only be used in the following config which shows a lot of redundancy:

    <extension target="org.nuxeo.runtime.avro" point="avroSchema">
      <avroSchema name="tutu" file="data/example.json" />
    </extension>

I would prefer the simpler:

    <extension target="org.nuxeo.runtime.avro" point="schema">
      <schema name="tutu" file="data/example.json" />
    </extension>

Reply:
If you would prefer then .. done